tools/breaking-changes/data/breaking-changes.json

```diff
@@ -23712,7 +23712,7 @@
         "apiElementKind": "Interface",
         "entryPoint": "@spartacus/checkout/occ",
         "deletedComment": "// TODO:Spartacus - PropertySignature 'paymentTypes' was removed from Interface 'CheckoutOccEndpoints'.",
-        "migrationComment": "",
+        "migrationComment": "By leveraging TS' type augmentation, it's part of the 'OccEndpoints'.",
```

Comment:
when b2b checkout is loaded (have to specify)

Reply:
Clarified